mp-prod/conf.d/health_metrics.yaml

```diff
@@ -390,6 +390,20 @@ health_metrics:
         weight: 1
       - expression: "css.api_down"
         weight: 2
+  ### DATAARTS
+  dataarts:
+    service: dataarts
+    component_name: "Data Arts Studio"
```

Comment:
please fix it to be either Data Arts Studio or Data Arts Service
but not 2 different names (health metric component name and service name in sdb_prod/catalog.yaml

Reply:
Fixed naming of DataArts.